test/firtool/firtool.fir

```diff
@@ -8,6 +8,22 @@ circuit test_mod :
     output b: UInt<1>
     b <= a
 
+  module Queue_1 :
```

Comment:
This test directory is supposed to be for the firtool command itself. Can you move the test to the test/firrtl/ directory in MLIR form?

Thanks!